spec/enumerable_spec.rb

```diff
@@ -0,0 +1,85 @@
+# spec/enumerables_spec.rb
+
+require_relative '../my_methods.rb'
+
+RSpec.describe Enumerable do
+  let(:test1) { [5, 4, 2] }
+  let(:test2) { [3] }
+  let(:test3) { [nil, true, 10] }
+  let(:test4) { [1, 2, 3, 1, 2, 3, 4, 1, 2, 3] }
```

Comment:
Please include other types of enumerable in your testing variables, like hashes or ranges.